app/assets/stylesheets/application.css

```diff
@@ -0,0 +1,258 @@
+  font-weight: bold;
+}
+
+#navbarSupportedContent {
+  display: flex;
+  align-items: center;
+}
+
+a {
+  text-decoration: none;
+}
+
+a:hover {
+  text-decoration: underline;
+}
+
+#signin-up-menu {
+  list-style: none;
+  display: flex;
+  margin: 0;
+  padding: 0;
+}
+
+li {
+  margin-bottom: 10px;
+}
+
+#signin-up-menu li {
+  margin-right: 15px;
+}
+
+#signin-up-menu a {
+  text-decoration: none;
+  color: white; /* Text color for sign-in/sign-up links */
+  font-weight: bold;
+}
+
+.btn-warning {
+  background-color: var(--second-color); /* Background color for the sign-out button */
+  color: white;
+  border: none;
+  padding: 8px 15px;
+  cursor: pointer;
+}
+
+/* Flash Messages */
+.alert-success {
+  background-color: var(--second-color); /* Background color for success messages */
+  color: white;
+  padding: 10px;
+  margin-top: 10px;
+}
+
+.alert-danger {
+  background-color: var(--second-color-dark); /* Background color for error messages */
+  color: white;
+  padding: 10px;
+  margin-top: 10px;
+}
+
+/* Container for the Splash Screen containing the Logo and the Sign-in/Sign-up Buttons */
+
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 80vh;
+}
+
+/* Customize this button as per design requirements */
+
+.button-login {
+  margin-top: 20px;
+  margin-bottom: 20px;
+  background-color: var(--main-color); /* Background color for the sign-in/sign-up buttons */
+  border: none;
+  padding: 20px 40px;
+  width: 100%;
+  border-radius: 5px;
+}
+
+.buttons-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  max-width: 500px;
+}
+
+/* make Everything in the center in categories container */
+
+.categories-container,
+.transactions-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+/* Make the Category Cards Style and Layout and Responsiveness */
+
+.category-card {
+  width: 100%;
+  max-width: 500px;
+  margin-bottom: 20px;
+  border-radius: 5px;
+  box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  padding: 10px;
+}
+
+img {
+  width: 50px;
+  height: 50px;
+  border-radius: 50%;
+  margin-right: 10px;
+}
+
+.transaction-card {
+  width: 100%;
+  max-width: 500px;
+  margin-bottom: 20px;
+  border-radius: 5px;
+  box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  padding: 10px;
+}
+
+ul {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.add-category-button,
+.add-transaction-button {
+  background-color: var(--second-color); /* Background color for the add category button */
+  color: white;
+  border: none;
+  padding: 8px 15px;
+  cursor: pointer;
+  border-radius: 5px;
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
```

Comment:
- [ ] I can see you are almost there but just a little push. Here is a suggestion on how to fix the button to the bottom of the page as requested by the previous reviewers. Simply add the following code to this button styling and modify the `padding` style to this.

  ```
  padding: 20px 15px;
  ...
  position:fixed;
  bottom: 0;
  ```

  ![image](https://github.com/Abdulwhbhussain/Budget-app/assets/36542986/8f19cc2c-7064-4b42-80f4-1db9266e68ad)

Reply:
Sure Bro!